src/components/ProfileCard/ProfileCard.styles.ts

```diff
@@ -0,0 +1,236 @@
+    boxShadow: theme.shadows[2],
+    [theme.breakpoints.down('sm')]: {
+      width: theme.spacing(6),
+      height: theme.spacing(6),
+      fontSize: theme.spacing(2.4),
+    },
+  };
+});
+
+export const StyledInfoContainer = styled(Box, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<StyledProfileCardProps>(({ theme, $variant }) => {
+  const variantConfig = VARIANT_CONFIGS[$variant];
+
+  return {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(0.5),
+    textAlign: variantConfig.textAlign,
+    flex: $variant === 'compact' ? 1 : 'initial',
+    minWidth: 0,
+    [theme.breakpoints.down('sm')]: {
+      textAlign: 'center',
+    },
+  };
+});
+
+export const StyledName = styled(Typography, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<{ $size: ProfileCardSize }>(({ theme, $size }) => {
+  const sizeConfig = SIZE_CONFIGS[$size];
+
+  return {
+    fontSize: sizeConfig.nameFontSize,
+    fontWeight: theme.typography.fontWeightBold || 600,
+    color: theme.palette.text.primary,
+    lineHeight: theme.typography.body1.lineHeight,
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+    [theme.breakpoints.down('sm')]: {
+      fontSize: theme.typography.body1.fontSize,
+    },
+  };
+});
+
+export const StyledUsername = styled(Typography, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<{ $size: ProfileCardSize }>(({ theme, $size }) => {
+  const sizeConfig = SIZE_CONFIGS[$size];
+
+  return {
+    fontSize: sizeConfig.usernameFontSize,
+    fontWeight: theme.typography.fontWeightRegular || 400,
+    color: greyScale[600],
+    lineHeight: theme.typography.body2.lineHeight,
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+    [theme.breakpoints.down('sm')]: {
+      fontSize: theme.typography.body2.fontSize,
+    },
+  };
+});
+
+export const StyledRole = styled(Typography, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<{ $size: ProfileCardSize }>(({ theme, $size }) => {
+  const sizeConfig = SIZE_CONFIGS[$size];
+
+  return {
+    fontSize: sizeConfig.roleFontSize,
+    fontWeight: theme.typography.fontWeightMedium || 500,
+    color: theme.palette.primary.main,
+    lineHeight: theme.typography.caption.lineHeight,
+    marginTop: theme.spacing(0.5),
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+    [theme.breakpoints.down('sm')]: {
+      fontSize: theme.typography.caption.fontSize,
+    },
+  };
+});
+
+export const StyledStatsContainer = styled(Box, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<StyledProfileCardProps>(({ theme, $size, $variant }) => {
+  const sizeConfig = SIZE_CONFIGS[$size];
+  const variantConfig = VARIANT_CONFIGS[$variant];
+
+  return {
+    display: 'flex',
+    flexDirection: variantConfig.statsDirection,
+    gap: theme.spacing(sizeConfig.statsGap / 8),
```

Comment:
The stats gap uses `theme.spacing(sizeConfig.statsGap / 8)` (again dividing by 8). This repeats the fragile pattern of storing px in constants and converting to spacing units at runtime.

**Sugerencia:**
```typescript
Store `statsGap` as spacing units in the constants and call `theme.spacing(sizeConfig.statsGapUnits)` here. This keeps spacing consistent with the theme and avoids magic arithmetic.
```